app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -35,14 +39,17 @@
     suggestion_link_text = intervention_helper.accessible_text
   end
 
+  data_attributes[:module] = "intervention"
+  data_attributes["intervention-name"] = name
```

Comment:
Nitpick: could these two lines setting `data_attributes` be moved up to around line 11, to put them together with the initial declaration for `data_attributes`?